interpretation/ww_head.py

```diff
@@ -202,8 +203,7 @@ def process(message, isGameMaster = False):
         '''myrole'''
         # This command sends the user's role back to them in a DM.
         if is_command(message,['myrole']):
-            # TODO
-            return todo()
+            return Mailbox().dm("Your role is **{}**.".format(db.db_get(message.author.id,'role')), message.author.id)
```

Comment:
I believe randium's code expects a string, and message.author.id is an int? Perhaps randium casts it to an int though.

Reply:
It shouldn't expect a str because I literally copied and pasted that from another function of mine

Reply:
As of now, the function is expecting an int, though I will go ahead now and cast it in main.py to an int anyway.

Reply:
Oh! Thing that does need to change, is you should remember to always return a list of a mailbox, not a mailbox itself.